packages/loot-core/src/server/accounts/parse-file.js

```diff
@@ -86,9 +86,7 @@ async function parseQIF(filepath) {
 }
 
 async function parseOFX(filepath) {
-  let { getOFXTransactions, initModule } = await import(
-    /* webpackChunkName: 'xfo' */ 'node-libofx'
-  );
+  let { getOFXTransactions, initModule } = await import('node-libofx');
```

Comment:
Does esbuild do code splitting automatically? This is a pretty hefty dependency that includes a bunch of wasm and js, so we’d prefer not to actually load it until the user requests an OFX file import.